### DISCUSSION TITLE
It seems like it only works for range between 0 and 1

### DISCUSSION THREAD

**Discussion by kalafun** [spacenation/swiftui-sliders#61] [Q&A] [answered]
I've tried your solution and it seems like it only works for range from 0 to 1. If I change it to be anything else above 1 the functionality behaves unpredictably. Is that so, or am I doing something wrong?

**Comment by ay42** [comment, marked as answer]
@kalafun Default bounds are 0 to 1 but you can set any range 
```
ValueSlider(value: $value, in: 30 ... 100)
```
